jupyterlab-server-proxy/README.md

```diff
@@ -2,31 +2,30 @@
 
 Launcher icons for proxied applications
 
-
 ## Prerequisites
 
-* JupyterLab
+- JupyterLab >=2,<3
```

Comment:
This confuses me significantly - as jupyterlab 3 support exists for this extension

Reply:
d9aff30